src/app/models/AnalysisJob.ts

```diff
@@ -96,7 +97,7 @@ export class AnalysisJob extends AbstractModel implements IAnalysisJob {
   }
 
   public get viewUrl(): string {
-    throw new Error("AnalysisJob viewUrl not implemented.");
+    return adminAnalysisJobMenuItem.route.format({ analysisJobId: this.id });
```

Comment:
incorrect. 

`viewUrl` should point to `/analysis_jobs/123` - the view normal users see
`adminViewUrl` could point to `/admin/analysis_jobs/123`

Reply:
Currently im using that as a placeholder so that components would crash as the analysis jobs route doesnt exist. I've added a TODO to highlight the issue